ui/src/app/settings/components/project-edit-panel/project-edit-panel.tsx

```diff
@@ -54,6 +55,13 @@ export const ProjectEditPanel = (props: {nameReadonly?: boolean; defaultParams?:
                     <div className='argo-form-row'>
                         <FormField formApi={api} label='Project Description' field='description' component={Text} />
                     </div>
+                    <React.Fragment>
+                        <div className='argo-form-row'>
+                            <h4>Labels</h4>
+                            <div>Labels are intended to be used to specify identifying attributes of project that are meaningful</div>
+                            <FormField formApi={api} field='labels' component={MapInputField} />
```

Comment:
Unable to use the component "MapInputField" correctly. When pressing "+" to add a new item, the form will flash out and submit directly.